### PR TITLE
Feature: Shift Click in Brewing Stand

### PR DESCRIPTION
**Pull request by Thunderblade73**

### CODE REVIEW

**Review by CalMWolfs — APPROVED** [review]
worked :)

File: src/main/java/at/hannibal2/skyhanni/features/inventory/ShiftClickBrewing.kt [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        if (!chestName.startsWith("Brewing Stand")) return
+
+        event.isCanceled = true
+        Minecraft.getMinecraft().playerController.windowClick(
+            event.container.windowId,
+            event.slot.slotNumber,
+            event.clickedButton,
+            1,
+            Minecraft.getMinecraft().thePlayer
+        )
```
Comment (lineargraph) [inline src/main/java/at/hannibal2/skyhanni/features/inventory/ShiftClickBrewing.kt]: This potentially sends an invalid packet if `event.clickedButton == 2`. Please always send `0` as button. See https://wiki.vg/index.php?title=Protocol&oldid=7368#Click_Window for valid modes / buttons.

**Review by lineargraph — CHANGES_REQUESTED** [review]
Not sure how needed this is with autobrewers being a thing. Aside from the merits of this PR, there is also a technical issue, pls fix.

File: src/main/java/at/hannibal2/skyhanni/features/inventory/ShiftClickBrewing.kt (hunk 1 above)
Comment (Thunderblade73) [reply src/main/java/at/hannibal2/skyhanni/features/inventory/ShiftClickBrewing.kt]: thy. I didn't notice the illegale state. Thought shift MMB was a thing 😅

**Review by lineargraph — APPROVED** [review]

**Review by hannibal002 — APPROVED** [review]

**Pull request merged**